lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -173,6 +173,25 @@ def scrub(transcript)
           gsub(%r((<cryptogram>).+(</cryptogram>)), '\1[FILTERED]\2')
       end
 
+      def supports_network_tokenization?
+        card = Billing::NetworkTokenizationCreditCard.new({
+          :number => "4111111111111111",
+          :month => 12,
+          :year => 20,
+          :first_name => 'John',
+          :last_name => 'Smith',
+          :brand => 'visa',
+          :payment_cryptogram => 'EHuWW9PiBkWvqE5juRwDzAUFBAk='
```

Comment:
> Is that cryptogram reusable?

Doesn't matter, we don't care about the success/failure response, we only care that the '155' error code wasn't returned (which means the processor doesn't support network tokenization)